src/mapnode.cpp

```diff
@@ -395,6 +395,77 @@ static std::vector<aabb3f> transformNodeBox(const MapNode &n,
 			boxes.push_back(box);
 		}
 	}
+	else if (nodebox.type == NODEBOX_CONNECTED)
+	{
+
```

Comment:
Must be the jetlag still, reserve space for? where? You mean cache this somehow?